docs/standard/generics/math.md

```diff
@@ -0,0 +1,199 @@
+
+### Numeric interfaces
+
+This section describes the interfaces in <xref:System.Numerics?displayProperty=fullName> that describe number-like types and the functionality available to them.
+
+| Interface name                                         | Description                                                                                            |
+|--------------------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| <xref:System.Numerics.IBinaryFloatingPointIeee754%601> | Exposes APIs common to *binary* floating-point types<sup>1</sup> that implement the IEEE 754 standard. |
+| <xref:System.Numerics.IBinaryInteger%601>              | Exposes APIs common to binary integers<sup>2</sup>.                                                    |
+| <xref:System.Numerics.IBinaryNumber%601>               | Exposes APIs common to binary numbers.                                                                 |
+| <xref:System.Numerics.IFloatingPoint%601>              | Exposes APIs common to floating-point types.                                                           |
+| <xref:System.Numerics.IFloatingPointIeee754%601>       | Exposes APIs common to floating-point types that implement the IEEE 754 standard.                      |
+| <xref:System.Numerics.INumber%601>                     | Exposes APIs common to comparable number types (effectively the "real" number domain).                 |
+| <xref:System.Numerics.INumberBase%601>                 | Exposes APIs common to all number types (effectively the "complex" number domain).                     |
+| <xref:System.Numerics.ISignedNumber%601>               | Exposes APIs common to all signed number types (such as the concept of `NegativeOne`).                 |
+| <xref:System.Numerics.IUnsignedNumber%601>             | Exposes APIs common to all unsigned number types.                                                      |
+| <xref:System.Numerics.IAdditiveIdentity%602>           | Exposes the concept of `(x + T.AdditiveIdentity) == x`.                                                |
+| <xref:System.Numerics.IMinMaxValue%601>                | Exposes the concept of `T.MinValue` and `T.MaxValue`.                                                  |
+| <xref:System.Numerics.IMultiplicativeIdentity%602>     | Exposes the concept of `(x * T.MultiplicativeIdentity) == x`.                                          |
+
+<sup>1</sup>The binary floating-point types are <xref:System.Double> (`double`), <xref:System.Half>, and <xref:System.Single> (`float`).
+
+<sup>2</sup>The binary integer types are <xref:System.Byte> (`byte`), <xref:System.Int16> (`short`), <xref:System.Int32> (`int`), <xref:System.Int64> (`long`), <xref:System.IntPtr> (`nint`), <xref:System.SByte> (`sbyte`), <xref:System.UInt16> (`ushort`), <xref:System.UInt32> (`uint`), <xref:System.UInt64> (`ulong`), and <xref:System.UIntPtr> (`nuint`).
```

Comment:
It's worth adding `Int128` and `UInt128` here:
```suggestion
<sup>2</sup>The binary integer types are <xref:System.Byte> (`byte`), <xref:System.Int16> (`short`), <xref:System.Int32> (`int`), <xref:System.Int64> (`long`), <xref:System.Int128>, <xref:System.IntPtr> (`nint`), <xref:System.SByte> (`sbyte`), <xref:System.UInt16> (`ushort`), <xref:System.UInt32> (`uint`), <xref:System.UInt64> (`ulong`), <xref:System.UInt128>, and <xref:System.UIntPtr> (`nuint`).
```

`System.Numerics.BigInteger` is also part of this group, but may not be as important to users.